internal/crypto/crypto.go

```diff
@@ -37,22 +37,3 @@ func GenerateOtp(digits int) (string, error) {
 	otp := fmt.Sprintf(expr, val.String())
 	return otp, nil
 }
-
-// generateOtpFromCharset generates a random n-length otp from a charset
-func generateOtpFromCharset(length int, charset string) (string, error) {
-	b := make([]byte, length)
```

Comment:
cc: @kangmingtay 

Removing for now since unused but can reinstate if we pick up and use nano id again